src/Phinx/Db/Table/Table.php

```diff
@@ -73,10 +73,12 @@ public function getOptions()
      * Sets the table options
      *
      * @param array $options The options for the table creation
-     * @return void
+     * @return $this
      */
-    public function setOptions(array $options)
+    public function setOptions(array $options): Table
     {
         $this->options = $options;
+
+        return $this;
```

Comment:
The other classes under this namespace all return their class instance at the conclusion of `setOptions`, so adding this for consistency.